packages/plugins/src/plugins/diagnostic-fix.plugin.ts

```diff
@@ -21,14 +23,14 @@ export class DiagnosticFixPlugin extends Plugin {
     let diagnostics = this.getDiagnostics(fileName, commentTag);
     let tries = diagnostics.length + 1;
 
-    this.logger?.debug(`Plugin 'DiagnosticAutofix' run on ${fileName}`);
```

Comment:
Why do you replace logger with debug?
Winston is used as CLI output https://github.com/rehearsal-js/rehearsal-js/blob/master/packages/cli/src/commands/upgrade.ts#L65-L67

Reply:
> Why do you replace logger with debug?
winston.logger is noisy and other than actually debugging an issue, in-which we would use the common `DEBUG` module. 

All other logging for Rehearsal should happen within Commander or throw. I can't think of any other uses?

Reply:
The transport layer has a `level` option to set the maximum output level.
We can use `info` as a default level and change to `debug` if `--verbose` option passed.